fedmsg.d/sync2jira.py

```diff
@@ -53,7 +53,9 @@
                 'token_auth': 'YOUR_JIRA_ACCESS_TOKEN',
             },
         },
-        'default_github_project_fields': {'storypoints': ('Estimate', 'customfield_12310243')},
+        'default_jira_fields': {
+            'storypoints': 'customfield_12310243',
+            },
```

Comment:
@webbnh @ralphbean @Zyzyx I am changing this configuration. This is not backward compatible. So, wherever sync2jira is running, the configuration needs to be changed. Existence of the old `default_github_project_fields` will not be a problem.

I made this change as there is no need to specify the GitHub field as it is not specific to a particular Jira instance. As these fields are unique to a particular Jira instance, it would be more appropriate to refer to them as `default_jira_fields`.

Here is a complete example:

```
"default_jira_fields": {
  "storypoints": "customfield_12310243"
},

"map": {
  "github": {
    "developerproductivity/open-connectors": {
      "project": "TEST",
      "component": "kernel",
      "sync": [
        "pullrequest",
        "issue"
      ],
      "issue_updates": [
        "comments",
        "github_project_fields"
      ],
      "github_project_fields": {
        "storypoints": {
          "gh_field": "Estimate"
        },
        "priority": {
          "gh_field": "Priority",
          "options": {
            "P0": "Blocker",
            "P1": "Critical",
            "P2": "Major",
            "P3": "Minor",
            "P4": "Optional",
            "P5": "Trivial"
          }
        }
      }
    }
  }
},
```

cc. @kpiwko @mattreid